assets/js/modules/analytics/datastore/settings.js

```diff
@@ -129,6 +137,19 @@ export async function submitChanges( registry ) {
 		dispatch( MODULES_ANALYTICS ).setProfileID( profile.id );
 	}
 
+	// If `ga4Reporting` AND `enableUA` are both enabled, we need to reset the
+	// property and profile IDs to ensure that the UA settings are not saved.
+	if ( ga4ReportingEnabled && ! isUAEnabled ) {
+		dispatch( MODULES_ANALYTICS ).resetPropertyAndProfileIDs();
+	}
```

Comment:
The code comment here mentions GA4 and "Enable UA" both being _enabled_, but the code here is checking for GA4 Reporting being enabled and UA being _disabled_. 🤔

I think the code logic is right, just the comment needs to be updated?

Reply:
Correct. Updated it.